pydatastructs/Queue.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function
+from __
+
+
+
+
+class Queue():
+	'''
+	Creation of Queue Data Type in python with list prebuilt data type
+	------------------------------------------------------------------
+
+	Parameter
+	==========
+	max_size:
+		 type-int.
+		 denotes the maximum size of queue.
+	
+
+	Raises
+	=======
+	No particular raisse till now.
+
+	Operations available:
+	=====================
+	1. initialisation of queue.
+	2. Check whether Queue is empty or not.
+	3. Appending elements into the queue on a first come first serve basis.
+	4. popping elements on first arrival basis.
+	5. detecting length of Queue.
+	6. Check whether queue is full or not.
```

Comment:
Not needed.

Reply:
thanks

Reply:
Will I have to write anything in the __init__.py file?

Reply:
Yes, see how stack is added to `__init__.py` file. Follow a similar pattern.